bin/ch/WScriptJsrt.cpp

```diff
@@ -490,9 +495,39 @@ bool WScriptJsrt::PrintException(LPCWSTR fileName, JsErrorCode jsErrorCode)
             LPCWSTR errorMessage = nullptr;
             size_t errorMessageLength = 0;
 
-            IfJsrtErrorFail(ChakraRTInterface::JsGetPropertyIdFromName(L"message", &messagePropertyId), false);
-            IfJsrtErrorFail(ChakraRTInterface::JsGetProperty(exception, messagePropertyId, &messageProperty), false);
-            IfJsrtErrorFail(ChakraRTInterface::JsStringToPointer(messageProperty, &errorMessage, &errorMessageLength), false);
+            LPCWSTR errorName = nullptr;
+            size_t errorNameLength = 0;
+
+            IfJsrtErrorFail(ChakraRTInterface::JsGetPropertyIdFromName(L"name", &namePropertyId), false);
```

Comment:
I don't think we need extracting `name` and `message` here. It is being done inside [JavascriptError:EntryToString](https://github.com/Microsoft/ChakraCore/blob/master/lib/Runtime/Library/JavascriptError.cpp#L215) that is called from `ChakraRTInterface::JsStringToPointer(messageProperty, &errorMessage, &errorMessageLength)`.

The way I would do is:

```C++
if(exception != nullptr) {
  errorMessage = ChakraRTInterface::JsStringToPointer(exception);
  
  // delete existing code that extracts 'message' propery from exception object

  if (jsErrorCode == JsErrorCode::JsErrorScriptCompile || ...) {
    if (jsErrorCode == JsErrorCode::JsErrorScriptCompile) {
     //  ...
     fwprintf(stderr, L"%ls\n\tat code...", errorMessage, ...); // note here we won't use errorTypeString anymore.
    } else {
     // existing code that extracts 'stack' property value and its type
     if(stackPropertyType != JsUndefined) {
       errorMessage = JsStringToPointer(stackProperty);
     }
     fwprintf(stderr, L"%ls\n", errorMessage);
    }
  } else {
    // ScriptException
    fwprintf(stderr, L"%ls\n", errorMessage);
  }
} else {
  errorTypeString = ConvertErrorCodeToMessage(jsErrorCode);
  fwprintf(stderr, L"Error : %ls\n", errorTypeString);
}
```